### DISCUSSION TITLE
Sonar ping comes from nav terminal not transducer.

### DISCUSSION THREAD

**Discussion by BlackMoons** [FakeFishGames/Barotrauma#13613] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When using a drone with sonar, controlled from the main ship I noticed that the main ship is the source of sonar noise when you use debugdraw and aitargets. 

Expected behavior: Transducer being used to be the source, so the drone makes noise when operated remotely with sonar on.

### Reproduction steps

Have a ship with a remote controlled drone
Move it away from the ship, turn on active sonar on the drone, via a nav console on the ship that is connected to the drones transducer via wifi.
use console commands:
debugdraw
aitargets

and observe drone makes no sonar noise, but the main ship does. 

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Already very fixed https://github.com/FakeFishGames/Barotrauma/issues/12496 
![barodev small](https://github.com/FakeFishGames/Barotrauma/assets/122838333/414f12a2-1680-445e-8bd9-ac74ab958584)

**Comment by BlackMoons** [comment]
Opps, My bad. Looking forward to spring update!

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/issues/12496
